src/services/codecovAI/useCodecovAIInstallation.spec.tsx

```diff
@@ -0,0 +1,110 @@
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query'
```

Comment:
Any taste for turning this into a vite file? Otherwise maybe we can create a followup ticket

Reply:
I think I ran into some minor issues, and this is a bit time sensitive. I'll give it another try Tuesday before merging. Otherwise, will make a followup.

Reply:
no worries! I get it